benchexec/tools/WitForJBMC.py

```diff
@@ -0,0 +1,32 @@
+import benchexec.tools.template
+import benchexec.result as result
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+ def executable(self, tool_locator):
+  return tool_locator.find_executable("Wit4JBMC.py")
+ def name(self):
+  return "Wit4JBMC"
+ def cmdline(self, executable, options, task, rlimits):
```

Comment:
This module does not override `version`, which means that users will not see in the result tables which tool version was benchmarked. This can be confusing and hinders reproducibility.

The best would be to add a parameter like `--version` to your tool and read and return the version number from the method `version`. You can call the utility method `self._version_from_tool` to do this easily.

(Note that SV-COMP actually requires a proper version number for each tool.)